CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## v229 (7/30/2021)
 
+* Default Ruby version is now 2.7.4 (https://github.com/heroku/heroku-buildpack-ruby/pull/1193)
```

Comment:
@schneems This changelog entry has been added under the existing `v229` release, rather than under the "unreleased" section - I've opened #1205 to fix it.

Reply:
This happened on a rebase perhaps? If we put effort into productizing our changelog check it would be nice to optionally enforce:

- Entry must be in the "unreleased" section (with optional over-ride)
- Entry dates in headers must be added with the current date (with optional over-ride)
- Maybe more

Thanks for spotting this!

Reply:
I missed it in my review, so my bad!